packages/components/organisms/f-header/stories/header.stories.js

```diff
@@ -58,11 +61,12 @@ export const HeaderComponent = () => ({
             :showOffersLink="showOffersLink"
             :showHelpLink="showHelpLink"
             :locale="locale"
+            :logoLinkDisabled="logoLinkDisabled"
```

Comment:
Correction after renaming prop:
```suggestion
            :isLogoLinkDisabled="logoLinkDisabled"
```

Reply:
🦅  eyed. Fixed now!